packages/backend/src/services/SavedChartsService/SavedChartService.ts

```diff
@@ -80,8 +80,11 @@ export class SavedChartService {
         const { organizationUuid, projectUuid } = savedChart;
         if (
             user.ability.cannot(
-                'update',
-                subject('SavedChart', { organizationUuid, projectUuid }),
+                'manage',
+                subject('ScheduledDeliveries', {
```

Comment:
Note that this method was intended to check chart access and is used in methods not related to scheduled delivery.

Reply:
ok I can create a separate check for scheduled deliveries then

Reply:
done